app/assets/stylesheets/application.scss

```diff
@@ -370,3 +370,78 @@ td > .btn:not(:last-of-type) {
 // .userway_buttons_wrapper {
 //   margin-bottom: 5rem;
 // }
+
+.btn-dark-orange {
+  background-color: #f2994a; /* Default button color */
+  border-color: #f2994a; /* Match border color */
+  color: #fff; /* White text color */
+}
+
+.btn-dark-orange:hover {
+  background-color: #e67e22; /* Darker orange on hover */
+  border-color: #e67e22;
+}
+
+.btn-darker-orange {
+  background-color: #e24829; /* Default button color */
+  border-color: #e24829;
+  color: #fff;
+}
+
+.btn-darker-orange:hover {
+  background-color: #c0392b; /* Darker shade on hover */
+  border-color: #c0392b;
+}
+
+.makerstore-content {
+  max-width: 500px; /* Replaces inline style */
```

Comment:
Do not put page specific classes in application.scss, make a separate stylesheet and target a section with an id